api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -338,6 +338,66 @@ class WorksServiceTest
         result.left.get shouldBe a[ElasticError]
       }
     }
+
+    // See: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html#simple-query-string-syntax
+    describe("simple query string syntax") {
+      it("uses none but PHRASE simple query syntax") {
+        val work = createIdentifiedWorkWith(
+          title =
+            "+a -title | with (all the simple) query~4 syntax operators in it*"
+        )
+
+        // prefix
```

Comment:
"* at the end of a term signifies a prefix query"

Reply:
Left over from somewhere else.